### PR TITLE
feat(devtool): added devtool support (fixes #19)

### LINKED ISSUE #19: devtool validator
We just need an enum of acceptable `devtool` options

### PR DESCRIPTION
**Pull request by xjamundx**

### CODE REVIEW

**Comment by kentcdodds** [comment]
Wish I had travis setup on this!

**Comment by xjamundx** [comment]
Yeah FYI I'm getting errors running `npm run validate` on `master`, but `npm run test` is fine

**Comment by kentcdodds** [comment]
Yeah, I think `nyc` is having trouble. I'm getting issues in repos all over the place (including at work).... :-(

File: src/validators/devtool/index.js [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+}
+const pragmas = new RegExp('^(#@|@|#)')
+
 export default {
   key: 'devtool',
-  validate: noop,
+  validate: validateEntry,
 }
 
+function validateEntry(val) {
```
Comment (kentcdodds) [inline src/validators/devtool/index.js]: `validateEntry` looks like copy/paste ;P

Comment (kentcdodds) [inline src/validators/devtool/index.js]: should probably be called `validateDevtool` ;-)

Comment (kentcdodds) [inline src/validators/devtool/index.js]: `¯\_(ツ)_/¯`

**Pull request merged**